src/test/java/utilities/KeywordUtil.java

```diff
@@ -1004,7 +1006,11 @@ public static boolean clickJS(By locator, String logStep) {
 
         return obj == null;
     }
-
+    public static boolean clickJS(By locator) {
```

Comment:
Please add comments for method description

Reply:
Updated as per comments